app/services/opd_service/reports/mental_health.rb

```diff
@@ -0,0 +1,55 @@
+class OPDService::Reports::MentalHealth
+
+  def find_report(start_date:, end_date:, **_extra_kwargs)
+    mental_health(start_date, end_date)
+  end
+
+  def mental_health(start_date, end_date)
+    type = EncounterType.find_by_name 'Outpatient diagnosis'
+    data = Encounter.where('encounter_datetime BETWEEN ? AND ?
+      AND encounter_type = ?
+      AND c.concept_id IN(9796,9795,9797,9798,467,9799,9800,9801,9802,9803,2719,9804,9805,9806,9807,9808,9809,10569,9811,15)',
```

Comment:
Use concept names so the code is readable instead of hard coding the numbers here